Tutorials/G-CNN Honeycomb.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>Remarks:</p><ul><li>I studied group theory so I know what abelian groups are... But I know many who do not. Maybe it's worth mentioning, or just putting a link to wikipedia? </li><li>Does the formula have a small type? Or maybe it's just this thing that renders it badly... it should be $\sum_{\langle I,j \rangle}...$</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>